game-core/src/test/java/games/strategy/engine/lobby/client/ui/LobbyGameTableModelTest.java

```diff
@@ -25,88 +28,98 @@
 import games.strategy.test.TestUtil;
 import games.strategy.util.Tuple;
 
-@ExtendWith(MockitoExtension.class)
-public class LobbyGameTableModelTest {
-
-  private LobbyGameTableModel testObj;
-
-  @Mock
-  private IMessenger mockMessenger;
-  @Mock
-  private IChannelMessenger mockChannelMessenger;
-  @Mock
-  private IRemoteMessenger mockRemoteMessenger;
-  @Mock
-  private ILobbyGameController mockLobbyController;
-
-  private Map<GUID, GameDescription> fakeGameMap;
-  private Tuple<GUID, GameDescription> fakeGame;
-
-  @Mock
-  private GameDescription mockGameDescription;
-
-  @Mock
-  private INode serverNode;
-
-  @BeforeEach
-  public void setUp() {
-    fakeGameMap = new HashMap<>();
-    fakeGame = Tuple.of(new GUID(), mockGameDescription);
-    fakeGameMap.put(fakeGame.getFirst(), fakeGame.getSecond());
-
-    Mockito.when(mockRemoteMessenger.getRemote(ILobbyGameController.GAME_CONTROLLER_REMOTE))
-        .thenReturn(mockLobbyController);
-    Mockito.when(mockLobbyController.listGames()).thenReturn(fakeGameMap);
-    testObj = new LobbyGameTableModel(true, mockMessenger, mockChannelMessenger, mockRemoteMessenger);
-    Mockito.verify(mockLobbyController, Mockito.times(1)).listGames();
-
-    MessageContext.setSenderNodeForThread(serverNode);
-    Mockito.when(mockMessenger.getServerNode()).thenReturn(serverNode);
-    TestUtil.waitForSwingThreads();
-    assertThat("games are loaded on init", testObj.getRowCount(), is(1));
+public final class LobbyGameTableModelTest {
+  @ExtendWith(MockitoExtension.class)
+  @Nested
+  final class RemoveAndUpdateGameTest {
```

Comment:
Had to break this up into separate fixtures to avoid a Mockito strictness warning due to the fixture-level mocks not being used by the new `formatBotStartTime()` test.